src/lint.ts

```diff
@@ -31,4 +31,4 @@ export interface ILinterOptions {
     rulesDirectory: string;
 }
 
-export {default as Linter} from "./tslint";
+export import Linter = require("./tslint");
```

Comment:
can we `import * as Linter` in this file as well, instead of `require()`? I'd like to turn on the `no-require-imports` rule...